### PR TITLE
vowpal-wabbit: Fix build for 8.7.0

### PR DESCRIPTION
**Pull request by ryan-robeson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
The latest release switched to CMake and now requires a git submodule to build.

### CODE REVIEW

File: Formula/vowpal-wabbit.rb [hunk 1]
```diff
@@ -12,23 +15,24 @@ class VowpalWabbit < Formula
...
-    ENV["AC_PATH"] = "#{HOMEBREW_PREFIX}/share"
-    system "./autogen.sh", "--prefix=#{prefix}",
-                           "--with-boost=#{Formula["boost"].opt_prefix}"
-    system "make"
-    system "make", "install"
+    # The project provides a Makefile, but it is a basic wrapper around cmake
+    # that does not accept *std_cmake_args.
+    # The following should be equivalent, while supporting Homebrew's standard args.
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
```
Comment (igas) [inline Formula/vowpal-wabbit.rb]: I think we should add skip for examples/tests

Comment (ryan-robeson) [reply Formula/vowpal-wabbit.rb]: Could you clarify? I'm not seeing `examples/tests` in the build logs or the final installation. Is `-DBUILD_TESTS=OFF` what you're referring to? 

If so, I'll add that.

Comment (igas) [reply Formula/vowpal-wabbit.rb]: Yep, that's the one. 👍

Comment (ryan-robeson) [reply Formula/vowpal-wabbit.rb]: Added. Thanks! Anything else?

**Comment by fxcoudert** [comment]
Thanks @ryan-robeson

**Pull request closed without merging**